creator-node/src/services/prometheusMonitoring/prometheusRegistry.js

```diff
@@ -1,32 +1,57 @@
 const PrometheusClient = require('prom-client')
 
 const {
-  NamespacePrefix,
-  Metrics,
-  MetricNames
+  NAMESPACE_PREFIX,
+  METRICS,
+  METRIC_NAMES
 } = require('./prometheus.constants')
 
 /**
  * See `prometheusMonitoring/README.md` for usage details
  */
 
-module.exports = class PrometheusRegistry {
+class PrometheusRegistry {
   constructor() {
     // Use default global registry to register metrics
     this.registry = PrometheusClient.register
 
-    // Expose metric names from class for access throughout application
-    this.metricNames = MetricNames
-
     // Ensure clean state for registry
     this.registry.clear()
 
     // Enable collection of default metrics (e.g. heap, cpu, event loop)
     PrometheusClient.collectDefaultMetrics({
-      prefix: NamespacePrefix + 'default_'
+      prefix: NAMESPACE_PREFIX + '_default_'
     })
 
-    createAllCustomMetrics(this.registry)
+    this.initStaticMetrics(this.registry)
+
+    // Expose metric names from class for access throughout application
+    this.metricNames = { ...METRIC_NAMES }
+
+    this.namespacePrefix = NAMESPACE_PREFIX
+
+    this.regexes = []
+  }
+
+  /**
+   * Creates and registers every static metric defined in prometheus.constants.js
+   */
+  initStaticMetrics(registry) {
```

Comment:
whats a static metric?

Reply:
a static metric is a metric that is hard-coded / defined in prometheus.constants.js

Reply:
yeh i know haha - i mostly meant to document this / not create a new term but its nbd, can leave it